cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -93,7 +93,7 @@ func RetrieveValidatedConfigInfo(cfg *kubeadmapi.JoinConfiguration) (*clientcmda
 		}
 		detachedJWSToken, ok := insecureClusterInfo.Data[bootstrapapi.JWSSignatureKeyPrefix+token.ID]
 		if !ok || len(detachedJWSToken) == 0 {
-			return nil, errors.Errorf("token id %q is invalid for this cluster or it has expired. Use \"kubeadm token create\" on the master node to creating a new valid token", token.ID)
+			return nil, errors.Errorf("token id %q is invalid for this cluster or it has expired. Use \"kubeadm token create\" on the control-plane node to creating a new valid token", token.ID)
```

Comment:
looks like we have a typo here:
> on the control-plane node to creating a new valid token

should be:
> on the control-plane node to create a new valid token

can you please fix that, and sorry for not catching it earlier?

Reply:
Ok @neolit123